client/src/RestaurantList/RestaurantCard.jsx

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import { Card } from 'antd';
+import React from 'react';
+import './RestaurantCard.scss';
+import type { Restaurant } from '../flowTypes';
+
+type Props = {
+  restaurant: Restaurant
+}
+
+export const RestaurantCard = (props: Props) => {
+  const { name, accepts10bis, rating } = props.restaurant;
+  return (<Card className="restaurant-card">
+    <div className="card-details">
+      <h3>{name}</h3>
+      {accepts10bis && <span className="ten-bis">10bis</span>}
+      <div>Rating: {Array.from({ length: rating }, (_, i) =>
```

Comment:
the rating can be zero? the rating can be 1.5?

Reply:
Rating can be either 1, 2, or 3. It's defined in `flowTypes.js`